### PR TITLE
PMM-11034: Add cases for max query length option

### PR DESCRIPTION
**Pull request by nazarovvi**
https://pmm.cd.percona.com/job/pmm2-ui-tests/25831/

### CODE REVIEW

**Review by vyarosh — APPROVED** [review]
Looks good to me

File: tests/configuration/verifyPMMSettingsPageFunctionality_test.js [hunk 1]
```diff
@@ -417,3 +417,13 @@ Scenario(
     }
   },
 );
+
+Scenario('PMM-T1401 Verify Percona Alerting wording in Settings @nazarov', async ({
```
[the remark this answers is not in the document]
Comment (puneet0191) [reply tests/configuration/verifyPMMSettingsPageFunctionality_test.js:421]: @nazarovvi please address

File: tests/QAN/pages/qanFiltersFragment.js [hunk 2]
```diff
@@ -89,12 +98,17 @@ module.exports = {
...
 
     I.waitForVisible(this.fields.filterBy, 30);
     I.fillField(this.fields.filterBy, filterName);
-    I.waitForVisible(filterToApply, 20);
+    await I.asyncWaitFor(async () => {
+      I.click(this.buttons.refresh);
+
+      return await I.grabNumberOfVisibleElements(filterToApply);
+    }, 60);
+    // I.waitForVisible(filterToApply, 20);
```
[the remark this answers is not in the document]
Comment (nazarovvi) [reply tests/QAN/pages/qanFiltersFragment.js:111]: Fixed

File: tests/configuration/verifyPMMSettingsPageFunctionality_test.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (nazarovvi) [reply tests/configuration/verifyPMMSettingsPageFunctionality_test.js:421]: Fixed

**Comment by nazarovvi** [comment]
@beatahandzelova confirm changes please

**Comment by puneet0191** [comment]
@nazarovvi please run the PR with changes on the job

**Review by vyarosh — APPROVED** [review]
Looks good to me

**Pull request merged**